BSPConvert.Lib/Source/MaterialConverter.cs

```diff
@@ -249,13 +252,13 @@ private void AppendShaderParameters(StringBuilder sb, Shader shader)
 				sb.AppendLine("\t$alphatest 1");
 				sb.AppendLine("\t$alphatestreference 0.5");
 			}
-			else if (shader.contents.HasFlag(Q3ContentsFlags.CONTENTS_TRANSLUCENT))
-				sb.AppendLine("\t$translucent 1");
 
-			if (flags.HasFlag(ShaderStageFlags.GLS_SRCBLEND_ONE | ShaderStageFlags.GLS_DSTBLEND_ONE) &&
-				(textureStage?.rgbGen != ColorGen.CGEN_IDENTITY || flags.HasFlag(ShaderStageFlags.GLS_SRCBLEND_DST_COLOR)))
+			if (shader.stages[0].flags.HasFlag(ShaderStageFlags.GLS_SRCBLEND_ONE | ShaderStageFlags.GLS_DSTBLEND_ONE)) // blendFunc add
```

Comment:
I don't think the shader is guaranteed to have any stages so you should check if the array is empty before accessing it.